stories/General/TagStory.tsx

```diff
@@ -1,81 +1,294 @@
 import * as React from 'react';
+import styled from 'styled-components';
 
 import StorybookComponent from '../StorybookComponent';
 
+import Divider from '../../src/General/Divider';
+import Heading from '../../src/General/Heading';
 import Tag from '../../src/General/Tag';
-import { CloseIcon } from '../../src/General/Icon/components';
-
-const props = {
-  Tag: [
-    {
-      name: 'theme',
-      type: 'string',
-      defaultValue: <code>grey</code>,
-      possibleValue: <code>red | blue | orange | green | black</code>,
-      require: 'no',
-      description: 'Sets theme of Badge.',
-    },
-    {
-      name: 'block',
-      type: 'boolean',
-      defaultValue: <code>false</code>,
-      possibleValue: <code>true | false</code>,
-      require: 'no',
-      description: 'Sets Badge into block style.',
-    },
-    {
-      name: 'outline',
-      type: 'boolean',
-      defaultValue: <code>false</code>,
-      possibleValue: <code>true | false</code>,
-      require: 'no',
-      description: 'Sets Badge into outline style.',
-    },
-  ],
+import {
+  PrimaryColor,
+  SecondaryColor,
+  Greyscale,
+} from '../../src/Style/Colors';
+import {
+  AddIcon,
+  CloseIcon,
+  TrashIcon,
+} from '../../src/General/Icon/components';
+
+const TagContainer = styled.span`
+  margin-right: 15px;
+`;
+
+const CustomizedTagStory = () => {
+  const ClickableCloseIcon = styled(CloseIcon)`
+    fill: ${Greyscale.grey};
+    cursor: pointer;
+
+    &:hover,
+    &:active {
+      fill: ${Greyscale.black};
+    }
+  `;
+
+  const AddTag = styled(Tag)`
+    border-color: ${SecondaryColor.lightgrey};
+
+    &:hover {
+      border-color: ${Greyscale.grey};
+    }
+
+    &:active {
+      border-color: ${Greyscale.black};
+    }
+  `;
+
+  const ResetTag = styled(Tag)`
+    color: ${PrimaryColor.glintsred};
+    background-color: rgba(236, 39, 43, 0.05);
+
+    &:hover {
+      background-color: rgba(236, 39, 43, 0.1);
+    }
+
+    &:active {
+      background-color: rgba(236, 39, 43, 0.2);
+    }
+  `;
+
+  return (
+    <>
+      <StorybookComponent
+        title="Tag"
+        code="import { Tag } from 'glints-aries'"
+        usage={`import { Tag, CloseIcon } from 'glints-aries'
+import styled from 'styled-components';
+
+const ClickableCloseIcon = styled(CloseIcon)\`
+  fill: ${Greyscale.grey};
+  cursor: pointer;
+
+  &:hover,
+  &:active {
+    fill: ${Greyscale.black};
+  }
+\`;
+<Tag endIcon={<ClickableCloseIcon onClick={() => alert('Clicked!')} />}/>}>Add</Tag>
+`}
+      >
+        <Heading>Customized Tags</Heading>
+        <TagContainer>
+          <Tag
+            endIcon={<ClickableCloseIcon onClick={() => alert('Clicked!')} />}
```

Comment:
is it possible to make the tag have and `onStartIconClick` `onEndIconClick`, so we don't need to style `ClickableCloseIcon`

Reply:
The `ClickableCloseIcon` is used to customize the icon color, not related to the click event.

Reply:
yes I mean we should not let user to customize icon color

Reply:
If we agree that we should  a variant prop, we can define it in aries.